### PR TITLE
Added Play button to CMainMenu and made a slight menu change

### PR DESCRIPTION
**Pull request by Halo14-**
Play button launches a local server with the same default settings used when Host Game is selected. 

These settings can be changed by navigating to the following location:-

`C:\Program Files (x86)\MTA San Andreas 1.6\server\mods\deathmatch`

screenshot

As I could not find the font used in the original menu item images, another was used. If the original font is known, please let me know and I'll update the image accordingly.

### CODE REVIEW

File: MTA10/mods/deathmatch/CClient.cpp [hunk 1]
```diff
@@ -159,7 +159,17 @@ int CClient::ClientInitialize ( const char* szArguments, CCoreInterface* pCore )
             char* szPass = strtok ( NULL, " " );
 
             // Are we supposed to launch the server and play locally?
-            if ( stricmp ( szArguments, "local" ) == 0 )
+			if (stricmp(szArguments, "play") == 0)
+			{
+				// Create clientgame
+				g_pClientGame = new CClientGame(true);
+
```
Comment (Halo14-) [inline MTA10/mods/deathmatch/CClient.cpp:166]: I also plan to implement functionality to join the previous server connected to and have this behavior the default, then on launch, if the button is pressed, ask the user whether they would like to reconnect to the server they last connected to the last time they had MTA:SA open and if they have never joined a server simply start a local server as-is the case in this commit.

**Comment by AlexTMjugador** [comment]
This looks good! :+1: 

However, I think that code identation doesn't follow MTA coding guidelines, which you can read in the Wiki. It seems that you are using tabs instead of spaces, and you don't leave any spaces between the function name and arguments. If you are willing that your commit gets merged, following the standards is a good idea.

**Comment by qaisjp** [comment]
Does the Play button do the same thing as Host Game > Start?

**Comment by Halo14-** [comment]
@AlexTMjugador Thanks! I've just committed new versions of the files I changed in the initial commit after a read through :)

@qaisjp yes, it does. I do plan to implement functionality where the Play button will attempt to connect to the server previously connected to and if not default to what it does now and perhaps randomly load a gamemode instead of freeroam.

**Comment by qaisjp** [comment]
Hmm. I'm not too convinced about this feature.

The "best experience" isn't typically on a local server alone - new users would be compelled to just click "Play" expecting something interesting to happen.

The Host Game menu is more explicit and a shorthand isn't required. I also feel like the "planned functionality" would be too unpredictable.

Can you explain the advantages of adding this button?

**Comment by Halo14-** [comment]
@qaisjp It's not to encourage the use of MTA for the purposes of playing alone, it's to quickly head into a server, locally hosted or not. I understand that the Host Game menu allows for customization of the experience, but sometimes people just want to enter into a game. 

> I also feel like the "planned functionality" would be too unpredictable.

After some testing it doesn't seem like there's any issues with it. By 'random gamemodes' I'm talking about race/play/deathmatch/tdm/ctf/briefcase.

Perhaps for somebody who has not run MTA before, they should be asked what it is they would like to play, be it enter into a random locally hosted game in which they can invite their friends or join a random server based on a tag/gamemode preference they choose?

**Comment by ifarbod** [comment]
> but sometimes people just want to enter into a game.

Isn't that for P2P games? MTA is server-client based.

**Comment by Halo14-** [comment]
I'll close this pull request if necessary.

**Comment by Halo14-** [comment]
@ifarbod Isn't which for P2P games? The 'sometimes people just want to enter into a game' meant just getting into a game as fast as possible.

I'll close this pull request if necessary.

**Pull request closed without merging**